docs/getting_started.rst

```diff
@@ -279,9 +279,8 @@ teracy-dev Git Clone and Vagrant Up
    ..  code-block:: bash
 
        $ cd ~/
-       $ git clone https://github.com/teracyhq/dev.git teracy-dev
+       $ git clone https://github.com/teracyhq/dev.git -b v0.6.0-a5 teracy-dev
        $ cd teracy-dev
-       $ git checkout develop
```

Comment:
shoud have `$ git checkout v0.6.0-a5` for visual impact, like I usually don't like to use `git clone url -b`